src/content/deployment/subctl/_index.en.md

```diff
@@ -19,6 +19,23 @@ reducing the possibility of mistakes during the process.
 
 {{< subctl-install >}}
 
+### Installing specific versions
+
+By default, https://get.submariner.io will provide the latest release for subctl, and hence Submariner. Specific versions can be
+requested by using the VERSION environment variable.
+
+Avalailable options are:
+  * `latest`  (default)
+  * `devel`
```

Comment:
Can add a note explaining what 'devel' is? something like "(latest merged code)"